Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.3
```

Comment:
We can't change the tools version as we need to keep support for 5.2 - does that block the PR?

Reply:
I believe 5.3 is require for the 

```
condition: .when(
                     platforms: [Platform.iOS, Platform.macOS, Platform.tvOS, Platform.watchOS]
                 )
```

If this is a blocker then i might look into how i could make changes to this package so that an additional package could provide the `NIOTransportServices` decency. (either through a subclass or a plugin like solution).  what do you think would be the best approach?

Reply:
Add a `Package@swift-5.2.swift` manifest which contains the old version and then that should work (we might need to wrap the new methods in a Swift version check as well)

Reply:
ok i will give that a go, should i also update the GitHub workflow so that it includes both versions of swift?

Reply:
@gwynne could you update the CI with all the new fun stuff?